Surface_mesh_shortest_path/examples/Surface_mesh_shortest_path/shortest_path_sequence.cpp

```diff
@@ -87,14 +82,11 @@ struct Print_visitor : public boost::static_visitor<> {
 int main(int argc, char** argv)
 {
   // read input polyhedron
-  Polyhedron_3 polyhedron;
+  Mesh polyhedron;
```

Comment:
If the name of the type changes, I think the name of the variable should be changed, too.